server/types.py

```diff
@@ -27,3 +29,31 @@ class Map(NamedTuple):
     id: int
     name: str
     path: str
+    weight: int
+
+    def get_map(self) -> "Map":
+        return self
+
+
+class NeroxisGeneratedMap(NamedTuple):
+    id: str
+    version: str
+    spawns: int
+    map_size_pixels: int
+    weight: int
+
+    def get_map(self) -> Map:
+        """
+            Generate a map name based on the version and parameters. If invalid parameters are specified
+            hand back None
+        """
+        seed_bytes = random.getrandbits(64).to_bytes(8, 'little')
```

Comment:
Oof. Sadly `randbytes` isn't added until 3.9.

Reply:
yeah it made me sad too